services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaController.kt

```diff
@@ -39,7 +38,7 @@ class HankeKayttajaController(
     private val featureFlags: FeatureFlags,
 ) {
 
-    @GetMapping(produces = [APPLICATION_JSON_VALUE])
+    @GetMapping("/hankkeet/{hankeTunnus}/kayttajat", produces = [APPLICATION_JSON_VALUE])
```

Comment:
produces = [APPLICATION_JSON_VALUE] can this be removed now? Referring to application.properties change.

Reply:
I removed all explicit produces tags and added a check for all GET request tests that the response content type is correct.